devtools/src/codegen/flexhtx/mod.rs

```diff
@@ -0,0 +1,121 @@
+use bimap::BiHashMap;
+use minidsp::formats::xml_config::Setting;
+use strong_xml::XmlRead;
+
+use super::spec::*;
+
+pub struct Target {}
+impl crate::Target for Target {
+    fn filename() -> &'static str {
+        "flexhtx.rs"
+    }
+
+    fn symbols() -> bimap::BiMap<String, usize> {
+        symbols()
+    }
+
+    fn device() -> Device {
+        device()
+    }
+}
+
+pub(crate) fn input(input: usize) -> Input {
+    Input {
+        gate: Some(Gate {
+            enable: format!("BM_DGain_{}_status", input + 1),
+            gain: Some(format!("BM_DGain_{}", input + 1)),
+        }),
+        meter: Some(format!("Meter_In_{}", input + 1)),
+        //meter_d: Some(format!("Meter_D_In_{}", input + 1)),
+        peq: vec![], //no input peq
+        /*bass_management: Some(Gate {
+            enable: format!("BM_DGain_{}_status", 3 + output),
+            gain: Some(format!("BM_DGain_{}", 3 + output)),
+        }),*/
+        routing: (0..7usize)
+            .map(|output| Gate {
+                enable: format!("BM_Mixer_{}_{}_status", input + 1, output + 1),
+                gain: Some(format!("BM_Mixer_{}_{}", input + 1, output + 1)),
+                //polarity: Some(format!("BM_Mixer_{input}_{output}_pol")),
+            })
+            .collect(),
+    }
+}
+
+pub(crate) fn output(output: usize) -> Output {
+    Output {
+        gate: Gate {
+            enable: format!("DGain_{}_0_status", output + 1),
+            gain: Some(format!("DGain_{}_0", output + 1)),
+        },
+        /*routing: (0..7usize) //input number
+        .map(|input| Gate {
+            enable: format!("Out_Mixer_{input}_{output}_status"),
+            gain: Some(format!("Out_Mixer_{input}_{output}")),
+            polarity: Some(format!("Out_Mixer_{input}_{output}_pol")),
+        })
+        .collect(),*/
+        meter: Some(format!("Meter_Out_{}", output + 1)), /*  (0..2usize)
+                                                          .map(|ind| Some(format!("Meter_{}_{}", 1 + output, ind)))
+                                                          .collect(),*/
+        //meter_d: Some(format!("Meter_D_Out_{}", output + 1)),
+        delay_addr: Some(format!("Delay_{}_0", 1 + output)),
+        invert_addr: format!("polarity_out_{}_0", 1 + output),
+        peq: (1..=10usize)
+            .rev()
+            .map(|index| format!("PEQ_{}_{}", output + 1, index))
+            .collect(),
+        xover: Some(Crossover {
+            peqs: [1, 5]
+                .iter()
+                .map(|group| format!("BPF_{}_{}", output + 1, group))
+                .chain(
```

Comment:
In addition to BM_BPF_1_1 , there is a BM_BPF_ALL_1 and BM_BPF_ALL_5.  Not sure how to handle those.

Reply:
Their XML files in the device console app has some comments around it, I think it's safe to ignore.
> 		<!-- Do not use BM_BPF_ALL, use BM_MIXER instead -->